esx_service/utils/vm_listener.py

```diff
@@ -34,6 +33,16 @@
 from pyVmomi import VmomiSupport, vim, vmodl
 # vim api version used - version11
 
+# Need this to capture hostd down exception.
+try:
```

Comment:
We should have a better way to detect python version, for example: http://sweetme.at/2013/10/21/how-to-detect-python-2-vs-3-in-your-python-script/